src/sentry/static/sentry/app/views/organizationIntegrations/integrationItem.jsx

```diff
@@ -41,9 +34,45 @@ export default class IntegrationItem extends React.Component {
               </Tooltip>
             )}
           </IntegrationName>
-          <DomainName style={style}>{integration.domainName}</DomainName>
-        </Box>
+          <DomainName>{integration.domainName}</DomainName>
+        </Labels>
       </Flex>
     );
   }
 }
+
+const IntegrationName = styled.div`
+  font-size: 1.6rem;
+`;
+
+const DomainName = styled.div`
+  color: ${p => p.theme.gray3};
+  font-size: 1.4rem;
+`;
+
+const ExpandedLabels = styled(p => <Box pl={1} {...p} />)`
```

Comment:
There's a mixed usage of `<Box>` with `display: flex` (why not use `<Flex>`) and also inline usage of `pl` but CSS of `flex-direction`

Reply:
5d0b991